### PR TITLE
use dynaminc imports to not require unnecessary deps

### PR DESCRIPTION
**Pull request by ronenlu**

### CODE REVIEW

File: atlas_provider_django/management/commands/atlas-provider-django.py [hunk 1]
```diff
@@ -101,17 +52,84 @@ def get_connection_by_dialect(dialect):
...
+                    return not self._is_limited_data_type(field)
+
+                def _supports_limited_data_type_defaults(self):
+                    return True
+
+            class MockMySQLDatabaseFeatures(MySQLDatabaseFeatures):
+                def __init__(self, *args, **kwargs):
+                    super().__init__(*args, **kwargs)
+
+                def has_native_uuid_field(self):
```
Comment (masseelch) [inline atlas_provider_django/management/commands/atlas-provider-django.py:94]: This is besically checking if the mysql driver is 8 or lower, right?

Comment (ronenlu) [reply atlas_provider_django/management/commands/atlas-provider-django.py:94]: This is true, but In order to determine the mysql version I need to call the DB (what I cannot do). so I mock this method

Comment (masseelch) [reply atlas_provider_django/management/commands/atlas-provider-django.py:94]: We should think about maybe adding the target version to the query string / arguments in addition to the dialect

Comment (ronenlu) [reply atlas_provider_django/management/commands/atlas-provider-django.py:94]: Thats good point, but lets discuss is in seperate PR

**Review by masseelch — APPROVED** [review]

**Pull request merged**